scylla/src/lib.rs

```diff
@@ -104,6 +104,8 @@ pub mod transport;
 
 pub(crate) mod utils;
 
+pub use utils::test_utils;
```

Comment:
This should not be a part of the main crate's public interface. Perhaps we can introduce yet another crate (ugh) just for test utilities, use this crate as a `dev-dependency` and never publish it.

Ref: https://stackoverflow.com/questions/44539729/what-is-an-idiomatic-way-to-have-shared-utility-functions-for-integration-tests

Reply:
IIUC, you suggest such an arrangement: (arrows signify dependency)
scylla (lib) -> scylla-test-utils -> scylla (integration tests) 
Is the above correct?

Reply:
If "A -> B" means "B depends on A", then yes.

Reply:
Ugh, but if we do this we won't be able to use the utilities in `scylla`'s unit tests, which is sad.

Perhaps it would work if we marked the module as `#[cfg(test)]`? If it shows up in documentation, we can also mark if with `#[doc(hidden)]` - this is also what one of the SO answers suggests.

Reply:
If we mark the module as `#[cfg(test)]`, then it will work only in that crate's unit tests. I've already checked that.

Reply:
So far no good solution for that, so for now I'm leaving the present solution.

Reply:
Addressed the problem by use of `#[doc(hidden)]` and a proper comment.